modal/config.py

```diff
@@ -194,6 +193,7 @@ class _Setting(typing.NamedTuple):
     "worker_id": _Setting(),  # For internal debugging use.
     "restore_state_path": _Setting("/__modal/restore-state.json"),
     "force_build": _Setting(False, transform=lambda x: x not in ("", "0")),
+    "traceback": _Setting(False, transform=lambda x: x not in ("", "0")),
```

Comment:
Maybe `show_tracebacks`?

Reply:
I'm basing this off the `RUST_BACKTRACE=1` environment variable